packages/ui/components/form/toggleGroup/ToggleGroup.tsx

```diff
@@ -34,58 +33,29 @@ const OptionalTooltipWrapper = ({
 };
 
 export const ToggleGroup = ({ options, onValueChange, isFullWidth, ...props }: ToggleGroupProps) => {
-  const [value, setValue] = useState<string | undefined>(props.defaultValue);
-  const activeRef = useRef<HTMLSpanElement>(null);
-
-  useEffect(() => {
-    if (value && onValueChange) onValueChange(value);
-  }, [value, onValueChange]);
-
   return (
     <>
       <RadixToggleGroup.Root
         type="single"
         {...props}
-        onValueChange={setValue}
+        onValueChange={onValueChange}
         className={classNames(
           "min-h-9 bg-muted border-default relative inline-flex gap-0.5 rounded-md border p-1",
           props.className,
           isFullWidth && "w-full"
         )}>
-        {/* Active toggle. It's a separate element so we can animate it nicely. */}
-        <span
```

Comment:
I decided to remove the animated background (for now?). It doesn't update properly when the current value changes, and it already took me quite some time to try to get it to work (it's implementation already changed a few times in the past weeks). I think it's not worth it to try to keep fixing it, so removed it for now. We can always later try and see if we can add it back in a better way.